protos/v1alpha1/audit_log_request.proto

```diff
@@ -48,6 +48,12 @@ message AuditLogRequest {
     // Reads of configuration data and all access to user data through public
     // APIs.
     DATA_ACCESS = 2 [(log_name) = "audit.abcxyz/data_access"];
+
+    // For audit logging system (vs. human) generated events.
+    SYSTEM_EVENT = 3 [(log_name) = "audit.abcxyz/system_event"];
```

Comment:
Philosophical - should we leave some room between system_event and the other kinds? I would image we might want system_event to be 50 or something.

Reply:
Good idea.